packages/kbn-optimizer/src/limits.ts

```diff
@@ -74,7 +87,13 @@ export function updateBundleLimits(log: ToolingLog, config: OptimizerConfig) {
   let yaml = `pageLoadAssetSize:\n`;
   for (const metric of metrics.sort((a, b) => a.id.localeCompare(b.id))) {
     if (metric.group === 'page load bundle size') {
-      yaml += `  ${metric.id}: ${number(metric.value + DEFAULT_BUDGET)}\n`;
+      const existingLimit = config.limits.pageLoadAssetSize?.[metric.id];
+      const newLimit =
+        existingLimit != null && existingLimit >= metric.value
+          ? existingLimit
+          : metric.value + DEFAULT_BUDGET;
+
+      yaml += `  ${metric.id}: ${number(newLimit)}\n`;
```

Comment:
I originally wanted to format the numbers with `_`-thousands-separators, but I get the feeling that we're not really going to be managing this file manually, so maybe it's better to just use `jsYaml.safeDump()`?